.github/workflows/scaffold-cron.yaml

```diff
@@ -19,4 +27,54 @@ jobs:
       apply: true
       workspace: ${{ matrix.environment }}
       working-dir: infrastructure/scaffold
-      role-to-assume: arn:aws:iam::760772947412:role/github-actions/gh-oidc
\ No newline at end of file
+      role-to-assume: arn:aws:iam::760772947412:role/github-actions/gh-oidc
+
+  post-scaffold:
+    needs: [scheduled-create]
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        uses: actions/checkout@v3
+      - 
+        name: configure aws credentials
+        uses: aws-actions/configure-aws-credentials@v3
+        with:
+          role-to-assume: ${{ env.ROLE_TO_ASSUME }}
+          role-session-name: cd
+          aws-region: ${{ env.AWS_REGION }}
+      -
+        name: configure kubefile
+        run: aws eks update-kubeconfig --name ${{ env.ENVIRONMENT }}-master-cluster
+
+      - # This step will be moved to ArgoCD later. I need it now to test ArgoCD deployment 
+        name: allow aws ${{ env.AWS_USER }} user to access eks control plane
+        run: |
+          kubectl patch configmap/aws-auth -n kube-system --type merge -p "$(cat <<EOF
+          data:
+            mapUsers: |
+              - userarn: arn:aws:iam::${{ env.AWS_ACCOUNT }}:${{ env.AWS_USER }}
+                username: ${{ env.AWS_USER }}
+                groups:
+                  - system:masters
+          EOF
+          )"
+      -
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.5.7
+
+      - # I'll improve this step if needed, but I dont want to create k8s deployments with terraform
+        name: Deploy ArgoCD into K8S
+        working-directory: ${{ ENV.K8S_DEPLOYMENTS_FOLDER }}
+        run: |
+          terraform init && terraform validate && terraform workspace select -or-create ${{ env.ENVIRONMENT }}
```

Comment:
```suggestion
          terraform init && terraform workspace select -or-create ${{ env.ENVIRONMENT }}
```

Reply:
Why should I remove the validate?

Reply:
Well, you're planning and applying below, So I wonder why the validate ?